nixos/modules/services/monitoring/beszel-agent.nix

```diff
@@ -0,0 +1,107 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.beszel-agent;
+in
+{
+  meta.maintainers = with lib.maintainers; [ BonusPlay ];
+
+  options.services.beszel-agent = {
```

Comment:
More of a question than a review: Would it be better to have options under services.beszel.agent instead of services.beszel-agent, and the same for the hub?

Reply:
I've pushed commits with this change, however they are not tested yet. I'll deploy them to my infra for testing soon-ish (tomorrow?)

Reply:
Amazing! I guess in the future a service option `services.beszel.enable` could enable the agent and hub in one go, if the default options allow it somehow.
But that can be left for a different PR :)